CHANGELOG.md

```diff
@@ -42,6 +43,8 @@ n/a
 
 - `PMG\Queue\Lifecycle\DelegatingLifecycle::fromArray`. Use `fromIterable`
   instead.
+- The `PMG\Queue\MessageTrait` has been deprecated. The behavior it provided (using
+  the fully qualified class name as the message name) is now the default.
```

Comment:
Unless the `MessageTrait` is actually removed, it should probably stay in `Changed` section.

Reply:
I was removed and added back with a deprecation warning. It's in the `Deprecations` section now just so those deprecations are visible.